### PR TITLE
Return status code 400 from function to indicate bad request

### PR DESCRIPTION
**Pull request by huksley**
Fixed 200 status code problem from #22

### CODE REVIEW

File: api/api.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def get_co2_estimate(
     else:
         return {
             "error": "Not enough information was provided to calculate CO2 estimate."
-        }
+        }, 400
```
Comment (MarkusShepherd) [inline api/api.py]: Even in Python it's best avoided to have different return types depending on your input etc. Or does this style come from connexion?

Comment (huksley) [reply api/api.py]: Maybe its better to wrap in some Response class? 
Indeed it is from Flask docs https://flask.palletsprojects.com/en/1.1.x/quickstart/#about-responses

```python
@app.errorhandler(404)
def not_found(error):
    return render_template('error.html'), 404
```

Comment (huksley) [reply api/api.py]: Always return result, status_code tuple

**Comment by huksley** [comment]
> I think we should refactor api.py a bit to decouple the function for connexion from what the handler needs.

Could be done in a separate PR, IMO

**Comment by huksley** [comment]
Rebase confilict fixed by #31

**Pull request closed without merging**